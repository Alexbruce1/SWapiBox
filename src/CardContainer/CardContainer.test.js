import React from 'react';
import { shallow } from 'enzyme';


// should receive props
// 