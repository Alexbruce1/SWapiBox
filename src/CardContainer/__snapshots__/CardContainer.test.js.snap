// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CardContainer Should match the snapshot 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <CardContainer
    characters={
      Array [
        Object {
          "Homeworld": "stewjon",
          "Name": "Luke",
          "Species": "Human",
          "population": "2",
        },
        Object {
          "Homeworld": "stewjon",
          "Name": "Luke",
          "Species": "Human",
          "population": "2",
        },
        Object {
          "Homeworld": "stewjon",
          "Name": "Luke",
          "Species": "Human",
          "population": "2",
        },
        Object {
          "Homeworld": "stewjon",
          "Name": "Luke",
          "Species": "Human",
          "population": "2",
        },
        Object {
          "Homeworld": "stewjon",
          "Name": "Luke",
          "Species": "Human",
          "population": "2",
        },
        Object {
          "Homeworld": "stewjon",
          "Name": "Luke",
          "Species": "Human",
          "population": "2",
        },
        Object {
          "Homeworld": "stewjon",
          "Name": "Luke",
          "Species": "Human",
          "population": "2",
        },
        Object {
          "Homeworld": "stewjon",
          "Name": "Luke",
          "Species": "Human",
          "population": "2",
        },
        Object {
          "Homeworld": "stewjon",
          "Name": "Luke",
          "Species": "Human",
          "population": "2",
        },
        Object {
          "Homeworld": "stewjon",
          "Name": "Luke",
          "Species": "Human",
          "population": "2",
        },
      ]
    }
  />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <Card
          person={
            Object {
              "Homeworld": "stewjon",
              "Name": "Luke",
              "Species": "Human",
              "population": "2",
            }
          }
        />,
        <Card
          person={
            Object {
              "Homeworld": "stewjon",
              "Name": "Luke",
              "Species": "Human",
              "population": "2",
            }
          }
        />,
        <Card
          person={
            Object {
              "Homeworld": "stewjon",
              "Name": "Luke",
              "Species": "Human",
              "population": "2",
            }
          }
        />,
        <Card
          person={
            Object {
              "Homeworld": "stewjon",
              "Name": "Luke",
              "Species": "Human",
              "population": "2",
            }
          }
        />,
        <Card
          person={
            Object {
              "Homeworld": "stewjon",
              "Name": "Luke",
              "Species": "Human",
              "population": "2",
            }
          }
        />,
        <Card
          person={
            Object {
              "Homeworld": "stewjon",
              "Name": "Luke",
              "Species": "Human",
              "population": "2",
            }
          }
        />,
        <Card
          person={
            Object {
              "Homeworld": "stewjon",
              "Name": "Luke",
              "Species": "Human",
              "population": "2",
            }
          }
        />,
        <Card
          person={
            Object {
              "Homeworld": "stewjon",
              "Name": "Luke",
              "Species": "Human",
              "population": "2",
            }
          }
        />,
        <Card
          person={
            Object {
              "Homeworld": "stewjon",
              "Name": "Luke",
              "Species": "Human",
              "population": "2",
            }
          }
        />,
        <Card
          person={
            Object {
              "Homeworld": "stewjon",
              "Name": "Luke",
              "Species": "Human",
              "population": "2",
            }
          }
        />,
      ],
      "className": "card-container",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": "0.24938418750328273",
        "nodeType": "class",
        "props": Object {
          "person": Object {
            "Homeworld": "stewjon",
            "Name": "Luke",
            "Species": "Human",
            "population": "2",
          },
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": "0.9384061903450405",
        "nodeType": "class",
        "props": Object {
          "person": Object {
            "Homeworld": "stewjon",
            "Name": "Luke",
            "Species": "Human",
            "population": "2",
          },
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": "0.12378411664526845",
        "nodeType": "class",
        "props": Object {
          "person": Object {
            "Homeworld": "stewjon",
            "Name": "Luke",
            "Species": "Human",
            "population": "2",
          },
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": "0.8239562030861747",
        "nodeType": "class",
        "props": Object {
          "person": Object {
            "Homeworld": "stewjon",
            "Name": "Luke",
            "Species": "Human",
            "population": "2",
          },
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": "0.7938361871496131",
        "nodeType": "class",
        "props": Object {
          "person": Object {
            "Homeworld": "stewjon",
            "Name": "Luke",
            "Species": "Human",
            "population": "2",
          },
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": "0.9674736265578945",
        "nodeType": "class",
        "props": Object {
          "person": Object {
            "Homeworld": "stewjon",
            "Name": "Luke",
            "Species": "Human",
            "population": "2",
          },
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": "0.7846309405472018",
        "nodeType": "class",
        "props": Object {
          "person": Object {
            "Homeworld": "stewjon",
            "Name": "Luke",
            "Species": "Human",
            "population": "2",
          },
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": "0.5191862232852544",
        "nodeType": "class",
        "props": Object {
          "person": Object {
            "Homeworld": "stewjon",
            "Name": "Luke",
            "Species": "Human",
            "population": "2",
          },
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": "0.7409975495179257",
        "nodeType": "class",
        "props": Object {
          "person": Object {
            "Homeworld": "stewjon",
            "Name": "Luke",
            "Species": "Human",
            "population": "2",
          },
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": "0.9944190634895047",
        "nodeType": "class",
        "props": Object {
          "person": Object {
            "Homeworld": "stewjon",
            "Name": "Luke",
            "Species": "Human",
            "population": "2",
          },
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <Card
            person={
              Object {
                "Homeworld": "stewjon",
                "Name": "Luke",
                "Species": "Human",
                "population": "2",
              }
            }
          />,
          <Card
            person={
              Object {
                "Homeworld": "stewjon",
                "Name": "Luke",
                "Species": "Human",
                "population": "2",
              }
            }
          />,
          <Card
            person={
              Object {
                "Homeworld": "stewjon",
                "Name": "Luke",
                "Species": "Human",
                "population": "2",
              }
            }
          />,
          <Card
            person={
              Object {
                "Homeworld": "stewjon",
                "Name": "Luke",
                "Species": "Human",
                "population": "2",
              }
            }
          />,
          <Card
            person={
              Object {
                "Homeworld": "stewjon",
                "Name": "Luke",
                "Species": "Human",
                "population": "2",
              }
            }
          />,
          <Card
            person={
              Object {
                "Homeworld": "stewjon",
                "Name": "Luke",
                "Species": "Human",
                "population": "2",
              }
            }
          />,
          <Card
            person={
              Object {
                "Homeworld": "stewjon",
                "Name": "Luke",
                "Species": "Human",
                "population": "2",
              }
            }
          />,
          <Card
            person={
              Object {
                "Homeworld": "stewjon",
                "Name": "Luke",
                "Species": "Human",
                "population": "2",
              }
            }
          />,
          <Card
            person={
              Object {
                "Homeworld": "stewjon",
                "Name": "Luke",
                "Species": "Human",
                "population": "2",
              }
            }
          />,
          <Card
            person={
              Object {
                "Homeworld": "stewjon",
                "Name": "Luke",
                "Species": "Human",
                "population": "2",
              }
            }
          />,
        ],
        "className": "card-container",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": "0.24938418750328273",
          "nodeType": "class",
          "props": Object {
            "person": Object {
              "Homeworld": "stewjon",
              "Name": "Luke",
              "Species": "Human",
              "population": "2",
            },
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": "0.9384061903450405",
          "nodeType": "class",
          "props": Object {
            "person": Object {
              "Homeworld": "stewjon",
              "Name": "Luke",
              "Species": "Human",
              "population": "2",
            },
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": "0.12378411664526845",
          "nodeType": "class",
          "props": Object {
            "person": Object {
              "Homeworld": "stewjon",
              "Name": "Luke",
              "Species": "Human",
              "population": "2",
            },
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": "0.8239562030861747",
          "nodeType": "class",
          "props": Object {
            "person": Object {
              "Homeworld": "stewjon",
              "Name": "Luke",
              "Species": "Human",
              "population": "2",
            },
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": "0.7938361871496131",
          "nodeType": "class",
          "props": Object {
            "person": Object {
              "Homeworld": "stewjon",
              "Name": "Luke",
              "Species": "Human",
              "population": "2",
            },
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": "0.9674736265578945",
          "nodeType": "class",
          "props": Object {
            "person": Object {
              "Homeworld": "stewjon",
              "Name": "Luke",
              "Species": "Human",
              "population": "2",
            },
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": "0.7846309405472018",
          "nodeType": "class",
          "props": Object {
            "person": Object {
              "Homeworld": "stewjon",
              "Name": "Luke",
              "Species": "Human",
              "population": "2",
            },
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": "0.5191862232852544",
          "nodeType": "class",
          "props": Object {
            "person": Object {
              "Homeworld": "stewjon",
              "Name": "Luke",
              "Species": "Human",
              "population": "2",
            },
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": "0.7409975495179257",
          "nodeType": "class",
          "props": Object {
            "person": Object {
              "Homeworld": "stewjon",
              "Name": "Luke",
              "Species": "Human",
              "population": "2",
            },
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": "0.9944190634895047",
          "nodeType": "class",
          "props": Object {
            "person": Object {
              "Homeworld": "stewjon",
              "Name": "Luke",
              "Species": "Human",
              "population": "2",
            },
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
